tests/integration_test.go

```diff
@@ -119,3 +119,101 @@ func Test_ConcurrentTransactionSubmission(t *testing.T) {
 		assert.Equal(t, uint64(1), rcp.Status)
 	}
 }
+
+func Test_CloudKMSConcurrentTransactionSubmission(t *testing.T) {
+	srv, err := startEmulator(true)
+	require.NoError(t, err)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer func() {
+		cancel()
+		srv.Stop()
+	}()
+
+	grpcHost := "localhost:3569"
+	emu := srv.Emulator()
+	service := emu.ServiceKey()
+
+	client, err := grpc.NewClient(grpcHost)
+	require.NoError(t, err)
+
+	time.Sleep(500 * time.Millisecond) // some time to startup
+
+	// create new account with Cloud KMS keys used for key-rotation
+	keyCount := 5
+	createdAddr, err := bootstrap.CreateCloudKMSMultiKeyAccount(
+		client,
+		service.Address,
+		"0xee82856bf20e2aa6",
+		"0x0ae53cb6e3f42a79",
+		service.PrivateKey,
+	)
+	require.NoError(t, err)
+
+	cfg := &config.Config{
+		DatabaseDir:           t.TempDir(),
+		AccessNodeHost:        grpcHost,
+		RPCPort:               8545,
+		RPCHost:               "127.0.0.1",
+		FlowNetworkID:         "flow-emulator",
+		EVMNetworkID:          types.FlowEVMTestNetChainID,
+		Coinbase:              eoaTestAccount,
+		COAAddress:            *createdAddr,
+		COACloudKMSProjectID:  "flow-evm-gateway",
```

Comment:
@sideninja We are going to need a test project, for running this test in a CI.

Reply:
cc @sjonpaulbrown

Reply:
@m-Peter after talking to @sjonpaulbrown I realised we don't have an existing integration of KMS in CI so running this as part of it would be hard. I suggest making this test manual, meaning it only runs when a flag is provided and contains the config for the test, similarly to how that is done in GO SDK test https://github.com/onflow/flow-go-sdk/blob/master/crypto/cloudkms/cloudkms_test.go

Reply:
Added a condition to skip this test, unless the proper env variable is present.
See https://github.com/onflow/flow-evm-gateway/pull/282/commits/c1efe604846ca63ccd461d84700bc584e92bb1f8